plugins/indexing/plugin.go

```diff
@@ -111,3 +132,9 @@ func main() {
 		GRPCServer: plugin.DefaultGRPCServer,
 	})
 }
+
+// Printing the error makes it easier to see what went wrong in the plugin output
+func abort(err error) {
+	fmt.Println(err)
+	panic(err)
+}
```

Comment:
Shouldn't this write the error to the log instead?

Reply:
Good one, the exception is the case where we're unable to open the log file so there is a little bit of duplication. Also not 100% sure whether this function variable is idiomatic Golang, but to me it made more sense than injecting the logger into the abort function. WDYT?

Reply:
Usually a logger has something like `Fatal()` (like here https://cs.opensource.google/go/go/+/master:src/log/log.go;drc=f09db2bb9331f4b31afb867173b1773e6494af27;l=282), but I see that this logger doesn't have that 😅 But I guess we can add a method to `logger.go`?